go.mod

```diff
@@ -34,7 +34,7 @@ require (
 	github.com/fsnotify/fsnotify v1.7.0
 	github.com/gballet/go-libpcsclite v0.0.0-20191108122812-4678299bea08
 	github.com/gofrs/flock v0.8.1
-	github.com/golang-jwt/jwt/v4 v4.5.1
+	github.com/golang-jwt/jwt/v4 v4.5.2
```

Comment:
Why is this required?

Reply:
It is not, this came from some clean up. Will address